tests/Fixer/LanguageConstruct/FunctionToConstantFixerTest.php

```diff
@@ -226,6 +226,34 @@ class B
                 null,
                 ['functions' => ['get_called_class']],
             ],
+            [
+                '<?php class Foo{ public function Bar(){ echo static::class  ; }}',
+                '<?php class Foo{ public function Bar(){ echo get_class( $This ); }}',
+                ['functions' => ['get_class_this']],
+            ],
+            [
+                '<?php class Foo{ public function Bar(){ echo static::class; get_class(1, 2); get_class($a); get_class($a, $b);}}',
+                '<?php class Foo{ public function Bar(){ echo get_class($this); get_class(1, 2); get_class($a); get_class($a, $b);}}',
+                ['functions' => ['get_class_this']],
+            ],
+            [
+                '<?php class Foo{ public function Bar(){ echo static::class /* 0 */  /* 1 */ ;}}',
+                '<?php class Foo{ public function Bar(){ echo \get_class( /* 0 */ $this /* 1 */ );}}',
+                ['functions' => ['get_class_this']],
+            ],
+            [
+                '<?php class Foo{ public function Bar(){ echo static::class; echo __CLASS__; }}',
+                '<?php class Foo{ public function Bar(){ echo \get_class((($this))); echo get_class(); }}',
+                ['functions' => ['get_class_this', 'get_class']],
```

Comment:
`'get_class'` is necessary here?

Reply:
this tests the scenario with both fix-cases and configuration to fix both, so I think it is necessary